packages/gestalt/src/TypeaheadInputField.js

```diff
@@ -113,15 +113,15 @@ const TypeaheadInputFieldWithForwardRef: React$AbstractComponent<
       ENTER: 0,
     };
     // Up Arrow
```

Comment:
nit: you can remove all these comments since the imported keyCodes communicate effectively

Reply:
good call